doc/source/api/index.rst

```diff
@@ -4,33 +4,34 @@
 API reference
 =============
 
-This section describes the core Pythonic interfaces for Twin Runtimes.
-Here, you can find the application programming interfaces for 
-using the core Twin Runtime SDK functionalities, as well as an example
-of higher level abstraction class to evaluate the Runtimes.
+This section describes the core Pythonic interfaces for twin runtimes.
+Here, you can find the APIs for using twin runtime functionalities and
+the higher-level abstraction class for evaluating twin runtimes.
 
-Twin Runtime SDK
-----------------
+Twin runtimes
+-------------
 
-Implementation of the :ref:`ref_index_api_sdk` class enabling access to the core
-Twin Runtime SDK functionalities.
+The :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class provides access to
+twin runtime functionalities. For a workflow example, see :ref:`ref_index_api_sdk`.
```

Comment:
@PipKat. Following your comment, I think we can better emphasize here the distinction. I suggest:
"The :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class provides access to all the twin runtime functionalities. It is the lowest level API of the twin runtime SDK. For a workflow example, see ..."
What do you think? Thanks

Reply:
@lboucin Perfect! Change made and incorporated.